scss/inc/_grid.scss

```diff
@@ -30,16 +32,18 @@
         margin-block-end: 12px;
     }
     .alpha {
-        margin-left: 0 !important
+        margin-left: 0 !important;

```

Comment:
Not sure what this element is, does it need to become `margin-inline-start`?
Also isn't it written 12 times into the CSS unnecessarily?

Reply:
Also don't know what it is, but let's change it.

Usually I'd prefer to change only minimal amount of styles, because this also affects RTL. (But maybe nobody uses RTL with CurretGen anymore)